website/docs/appflow/native/getting-started.md

```diff
@@ -0,0 +1,157 @@
+    data-zoom-src={useBaseUrl("/img/create-a-new-build.webp")} 
+    width="100%"
+  />
+</em>
+
+<em style={{ textAlign: 'center', display: 'block'}}>
+  <img 
+    src={useBaseUrl("/img/create-a-new-build-selected.webp")} 
+    data-zoom-src={useBaseUrl("/img/create-a-new-build-selected.webp")} 
+    width="100%"
+  />
+</em>
+
+When creating a new build, Appflow will present a few configuration options.
+- **Target platform** should be **Web**.
+- **Build stack** should be the default latest build stack.
+- **Live update** should be toggled on and the selected channel should be **Production**. This can also be configured after the build is complete.
+
+Once you click **Build**, Appflow will spin up a new build runner that will install your project dependencies and build your web application. After the build is complete, it will deploy the build to our **Production** Live Update channel.
+
+With the application build and deployed, you can:
+1. Use the Portals CLI to pull down the web bundle during development & the native build process.
+2. Add the Live Updates SDK into your app.
+
+## Seeding the Portal at build time
+
+The [Portals CLI](../../cli/overview.md) offers some common functionality for native developers, including the ability to pull down the appropriate web bundles from Appflow using a single command.
+
+Begin by following the [installation instructions](../../cli/overview.md#installation) and ensuring the CLI is installed.
+
+### Creating a Personal Access Token
+
+The CLI needs a means to authenticate with Appflow. To do this, we can create a Personal Access Token from the [Personal Settings menu](https://dashboard.ionicframework.com/settings/personal-access-tokens). Click the **Generate new token** button and optionally provide an expiration date for the token.
+
+<em style={{ textAlign: 'center', display: 'block'}}>
+  <img 
+    src={useBaseUrl("/img/personal-access-token.webp")} 
+    data-zoom-src={useBaseUrl("/img/personal-access-token.webp")} 
+    width="100%"
+  />
+</em>
+
+:::tip
+After the token is generated you will need to copy it to clipboard because it will be required for the next step. The token follows the format of `ion_XXXXXXXXXXXXX`.
+:::
+
+### Configuring the Portals CLI
+
+In our native project, we will create a `.portals.yaml` file. This will contain a list of the apps we want to pull down from Appflow. In this example, we only have our demo application, but we can add as many apps as we'd like.
+
+There are two key components for this configuration. The `sync` value will be a list of apps with their respective Appflow app IDs, Live Update channel names, and the file path the bundle should be copied to. The `token` value will the Personal Access Token generated in the previous step. Make sure to replace the app ID, directory name, and token with the appropriate values. 
+
+```yaml
+sync:
+  - app-id: 11a0971f
+    channel: production
+    directory-name: path/to/web/bundle
```

Comment:
I would change this to `portals/debug` as this is a tutorial. I wouldn't want customers to think that `path/to/web/bundle` is a path they should realistically use.